src/components/Form/Form.tsx

```diff
@@ -0,0 +1,45 @@
+// HorizontalRule
+import * as React from "react";
+import bem from "../../utils/bem";
+import generateUUID from "../../helpers/generateUUID";
+
+export interface FormProps {
+  /** Optional additional attributes passed to the `<form>` element */
+  attributes?: { [key: string]: any };
+  /** Optional className you can add in addition to `form` */
+  className?: string;
+  /** Optional ID that other components can cross reference */
+  id?: string;
+  /** Optional modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+  modifiers?: string[];
+}
+
+// FormRow child-component
+export function FormRow({ ...props }) {
+  const { children, className } = props;
+  return <div className={bem("form-row", [], "", [className])}>{children}</div>;
+}
+
+// FormField child-component
+export function FormField({ ...props }) {
+  const { children, className } = props;
+  return (
+    <div className={bem("form-field", [], "", [className])}>{children}</div>
+  );
+}
+
+export default function Form(props: React.ComponentProps<"form"> & FormProps) {
+  const { attributes = {}, children, className, id, modifiers = [] } = props;
```

Comment:
We'll have to discuss this later but in case someone forgets to use the `attributes` prop and adds props directly to the component, this would need a `...rest` variable to catch those props.

Reply:
@EdwinGuzman I will create a new ticket for this functionality.  For now, I will release this as a basic UI component.

Reply:
I changed my mind and added props for `action` and `method`.